custom_components/solarman/solarman.py

```diff
@@ -166,6 +166,9 @@ def send_request(self, params, start, end, mb_fc, sock):
             else:
                 log.debug(f"Querying [{start} - {end}] failed, invalid response packet.")
             del raw_msg
+        except:
+            log.error("An exception was thrown!")
```

Comment:
Mind that since https://github.com/StephanJoubert/home_assistant_solarman/pull/174/files#diff-7df232aac98730fa627674270cfabb61505fb902d8a79c5efc4c0fb56c4502b1R213 any exception thrown here is covered by the retry mechanism in `get_statistics(self)` already (and intentionally was downgraded to `debug` level as transient connection errors seem to be rather the norm than the exception.

I believe this change was done with some old main branch in mind, i.e. https://github.com/StephanJoubert/home_assistant_solarman/blob/e7f2c4c01a6bec5be74da858a33d4d81d87f179a/custom_components/solarman/solarman.py#L170-L171

Reply:
Oha https://github.com/StephanJoubert/home_assistant_solarman/commit/f489ad70bbd1de3c93f5bea1b4d33a793956ec50 :bulb: nothing to see here :wink: